### PR TITLE
287 percentage calculation might not be correct

### LINKED ISSUE #287: Percentage calculation might not be correct
image
I think the percentage is meant for out of the same building cluster (row), what's the percentage of each damage state. E.g. the first row should be `10(66.67), 5(33.33)`
We might also default the percentage to be off.

### PR DESCRIPTION
**Pull request by ylyangtw**
- Fix percentage calculation
- Default `View Percentage` to false
image

### CODE REVIEW

File: src/components/childComponents/table/ResultTable.jsx [hunk 1]
```diff
@@ -84,19 +84,25 @@ function calcClusterNumInCategory(fields) {
 	return count;
 }
 
-function calcPercentage(partialValue, totalValue) {
+function calcPercentage(partialValue, categoryItem, tableHeaders) {
```
Comment (longshuicy) [inline src/components/childComponents/table/ResultTable.jsx]: I think it depends on situation, sometimes it's categoryItem, other times it should be clusterItem

**Review by longshuicy — CHANGES_REQUESTED** [review]
Some of the rows are not quite right.
e.g.
the total row ==> everything is 100% 
business - manufacture row everything is under 1%
I believe each row should add up to 100%

image

**Comment by ylyangtw** [comment]
Have another fix. Should be good now:
image

**Review by longshuicy — APPROVED** [review]
Looks good approve.

**Pull request merged**